src/App.jsx

```diff
@@ -6,18 +6,19 @@ import Tracks from "./pages/Tracks/Tracks.jsx";
 import Bikers from "./pages/Bikers/Bikers.jsx";
 import Error from "./pages/Error/Error.jsx";
 import Navbar from "./components/Navbar/Navbar";
+import Footer from "./components/Footer/Footer";
 
```

Comment:
Mishä te has pasado en la line 9, la proxima vez ves con más cuidad.
**No quiero probelmas en la casa!**
```suggestion
<footer></footer>
```
Mamishka is Cool Developer!